inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcBatchingOutputFormat.java

```diff
@@ -120,20 +131,29 @@ static JdbcStatementBuilder<Row> createRowJdbcStatementBuilder(int[] types) {
     public void open(int taskNumber, int numTasks) throws IOException {
         super.open(taskNumber, numTasks);
         this.runtimeContext = getRuntimeContext();
-        metricData = new MetricData(runtimeContext.getMetricGroup());
+        sinkMetricData = new SinkMetricData(runtimeContext.getMetricGroup());
         if (inLongMetric != null && !inLongMetric.isEmpty()) {
-            String[] inLongMetricArray = inLongMetric.split("&");
-            String groupId = inLongMetricArray[0];
-            String streamId = inLongMetricArray[1];
+            String[] inLongMetricArray = inLongMetric.split(DELIMITER);
+            inLongGroupId = inLongMetricArray[0];
+            inLongStreamId = inLongMetricArray[1];
             String nodeId = inLongMetricArray[2];
-            metricData.registerMetricsForDirtyBytes(groupId, streamId, nodeId, "dirtyBytes");
-            metricData.registerMetricsForDirtyRecords(groupId, streamId, nodeId, "dirtyRecords");
-            metricData.registerMetricsForNumBytesOut(groupId, streamId, nodeId, "numBytesOut");
-            metricData.registerMetricsForNumRecordsOut(groupId, streamId, nodeId, "numRecordsOut");
-            metricData.registerMetricsForNumBytesOutPerSecond(groupId, streamId, nodeId, "numBytesOutPerSecond");
-            metricData.registerMetricsForNumRecordsOutPerSecond(groupId, streamId, nodeId,
+            sinkMetricData.registerMetricsForDirtyBytes(inLongGroupId, inLongStreamId,
+                    nodeId, "dirtyBytes");
+            sinkMetricData.registerMetricsForDirtyRecords(inLongGroupId, inLongStreamId,
+                    nodeId, "dirtyRecords");
+            sinkMetricData.registerMetricsForNumBytesOut(inLongGroupId, inLongStreamId,
+                    nodeId, "numBytesOut");
+            sinkMetricData.registerMetricsForNumRecordsOut(inLongGroupId, inLongStreamId,
+                    nodeId, "numRecordsOut");
+            sinkMetricData.registerMetricsForNumBytesOutPerSecond(inLongGroupId, inLongStreamId, nodeId,
+                    "numBytesOutPerSecond");
+            sinkMetricData.registerMetricsForNumRecordsOutPerSecond(inLongGroupId, inLongStreamId, nodeId,
```

Comment:
maybe use connector-base string of Constans class

Reply:
@gong DONE